.github/workflows/build_and_deploy.yml

```
@@ -13,10 +13,16 @@ on:
     types:
       - published
   pull_request:
+    paths:
+      - build_util/**
```

参考: <https://github.com/VOICEVOX/voicevox_core/pull/345>

```suggestion
    paths:
      - .github/workflows/build_and_deploy.yml
      - build_util/**
```